File: src/main/java/ru/dragonestia/apocalypse/item/SulfurDustItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class SulfurDustItem extends Item implements AdvItem {

    public SulfurDustItem(Integer meta, int count) {
        super(ApocalypseID.SULFUR_DUST, meta, count, "Sulfur dust");
    }

    @Override
    public String getRuName() {
        return "Сера";
    }

    @Override
    public String getIcon() {
        return "textures/items/dust/sulfur";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/SulfurDustItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class SulfurDustItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.SULFUR_DUST, 0, count, "Sulfur dust"), AdvItem {

    override fun getRuName(): String = "Сера"

    override fun getIcon(): String = "textures/items/dust/sulfur"

}